JudoPayDotNet/Models/RegisterCardModel.cs

```diff
@@ -10,34 +10,8 @@ namespace JudoPayDotNet.Models
     [DataContract]
     [KnownType(typeof(RegisterEncryptedCardModel))]
     // ReSharper disable UnusedMember.Global
-    public class RegisterCardModel : IModelWithHttpHeaders
+    public class RegisterCardModel : ThreeDSecureTwoPaymentModel
```

Comment:
Unfortunately we can't make RegisterCardModel extend ThreeDSecureTwoPaymentModel as some of the base attributes of PaymentModel (e.g. UserAgent) are not defined in RegisterCardModel.

Instead the attributes from ThreeDSecureTwoPaymentModel can just be added into RegisterCardModel as well